src/components/Toolbar/Toolbar.tsx

```diff
@@ -70,6 +70,11 @@ const Toolbar: FC<ToolbarProps> = ({ onButtonClick }) => {
           text={t("common.join")}
           onClick={() => onToolbarButtonClick("join")}
         />
+        <ToolbarButton
+          iconName="learn"
+          text="Whitepaper"
```

Comment:
Add a translation for whitepaper with `useTranslation`. Ask Don or Brigand to help you add it to poeditor.

Also the word whitepaper doesn't fit. Please contact design how to solve this.
![Screenshot 2022-01-31 at 16 53 54](https://user-images.githubusercontent.com/86911296/151829233-73a016a8-8d6b-47d2-942b-c6962680e1d5.png)

Reply:
No need for toolbar button anymore. So this is resolved.